docs/source/program_overview.rst

```diff
@@ -0,0 +1,62 @@
+Program Overview
+================
+
+The general capabalities available through QCEngine for each program can be
+found below:
+
+Quantum Chemistry
+-----------------
+
++------------+------------+------------+-------------+--------------+
```

Comment:
Very happy to have this table.

Reply:
Me too! Although would it make sense to have separate check marks for E/G/H since I don't think Molpro currently has Hessian support through QCEngine?

Reply:
@sjrl Good point and we have the room.